tests/cut-plane/test_detection.py

```diff
@@ -74,16 +74,17 @@ def test_minimize():
                      -2.71528635e-04, 6.88986409e+01])
 
 
-@pytest.mark.skipif(version.parse(scipy.__version__) < version.parse('1.6'),
-                    reason="requires scipy >= 1.6")
-def test_minimize():
+@pytest.mark.skipif(version.parse(scipy.__version__) < version.parse('1.7'),
+                    reason="requires scipy >= 1.7")
+def test_minimize_more17():
```

Comment:
@adrien-berchet and @arnaudon I want to emphasize your attention here. Scipy 1.7.0 brought a major change: `scipy.optimize.minimize` applies `bound` argument for "Nelder-Mead" method. That's why we see here this change of expected data. It might affect your production data. The link to the change https://github.com/scipy/scipy/commit/f8cbb52188230328dc4a0c9857f5f14556ca380e#diff-61ffae76b40123d1acc8d871de22c1fbebdd49a8c1d90a7933e985a66a9f19a8